### PR TITLE
sync: preset urls and pins (fixes #3742)

### LINKED ISSUE #3742: easy settings configuration
preset addresses and pins for easy configurations and sync

### PR DESCRIPTION
**Pull request by Okuro3499**
fixes #3742
[Screen_recording_20240701_172320.webm](https://github.com/open-learning-exchange/myplanet/assets/64019708/75f8d189-760b-4fd5-9584-503a61e8e663)

### CODE REVIEW

File: app/src/main/java/org/ole/planet/myplanet/ui/sync/SyncActivity.kt [hunk 1]
```diff
@@ -588,14 +588,78 @@ abstract class SyncActivity : ProcessUserDataActivity(), SyncListener, CheckVers
...
     }
 
+    private fun getPinForUrl(url: String): String {
+        val pinMap = mapOf(
+            "planet.earth.ole.org" to "7379",
+            "planet.learning.ole.org" to "1983",
+            "planet.vi.ole.org" to "0660",
+            "planet.somalia.ole.org" to "5932",
+            "planet.guatemala.ole.org" to "5562",
+            "planet.sanpablo.ole.org" to "0948",
```
Comment (dogi) [inline app/src/main/java/org/ole/planet/myplanet/ui/sync/SyncActivity.kt:651]: 192.168.48.253

**Comment by Okuro3499** [comment]
Screenshot_20240702_185757

**Review by Mutugiii — APPROVED** [review]
Was able to switch between the Planets good. Sync takes time on my side

**Comment by dogi** [comment]
#3982

**Pull request closed without merging**